Include GraphQL teasers in relevant REST API intros (#149)
Trackunit/developer-hub: api-docs/location-api-intro.md

With two convenient endpoints at your disposal, you can choose to retrieve precise location snapshots for a specific asset or access a paginated list containing all your assets.
Perfect to sync latest asset location data to any other system.

> 📘 Also access location data for an Asset via the GraphQL API
> 

> > Via the Public GraphQL API, you can utilize query capabilities to fetch 'location' data connected to an asset. Explore the GraphQL schema through our [GraphQL Explorer](https://apps.iris.trackunit.com/graphql-public-viewer/). Follow the 'asset' entry point and select 'locations'. 

Reviewer: Maybe:
```suggestion
> Via the Public GraphQL API, you can utilize query capabilities to fetch 'location' data together with other asset data in one API call. Explore the GraphQL schema through our [GraphQL Explorer](https://apps.iris.trackunit.com/graphql-public-viewer/). Follow the 'asset' entry point and select 'locations'. 
```
